src/hw-3/services/user.sevice.ts

```diff
@@ -0,0 +1,60 @@
+import { NewUser, User } from '../types/user';
+import { Sequelize, DataTypes, Op } from 'sequelize';
+import { UserModel } from '../models/user';
+
+const sq = new Sequelize('postgres://postgres:genya147@localhost:5433/user_db');
```

Comment:
be honest I don't like that it's hardcoded. I'd like if you used `process.env` here to configure postgres username/password/host/port/db. As there are a lot of params, `dotenv` would work even better.

Reply:
Fixed. `dotenv` has been added, .env has been created.